rasa/cli/utils.py

```diff
@@ -40,10 +41,8 @@ def get_validated_path(
             if current is None:
                 reason_str = "Parameter '{}' not set.".format(parameter)
             else:
-                logger.warning(
-                    "'{}' does not exist. Using default value '{}' instead.".format(
-                        current, default
-                    )
+                warnings.warn(
```

Comment:
I'm actually not sure about this one, could be there are some cases where our default parameters in the CLI are `None` and we are inserting the default value here.

Reply:
Didn't know that. I think ideally we would differentiate between these cases. 
If it is possible that this is something the user should fix it still should be a warning, imo it's better to risk drawing attention to something that doesn't need to fixed than to risk not notifying a user about something they do need to fix

Reply:
Ok, just checked the code and it should be fine 👍